.sass-lint.yml

```diff
@@ -1,6 +1,8 @@
 # sass-lint config to match the AirBNB style guide
 files:
   include: '**/client/src/**/*.scss'
+  ignore:
+    - 'vendor/silverstripe/admin/*'
```

Comment:
This seems a bit odd to me... shouldn't it _exclusively_ be linting the files from this module, not "all client folders everywhere except admin"?

Reply:
Will remove, js job (which runs sass-lint) is broken even with this change.